Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM python:alpine3.12
+FROM python:3.10.0-slim
+
 COPY . /app
 WORKDIR /app
+
 RUN pip install -r requirements.txt
 CMD ["uvicorn", "--host", "0.0.0.0", "--port", "3001", "opensight_restserver:app"]
```

Comment:
Maybe we want to add `"--workers", "4"` here or in the docker-compose deployment.
So we don't forget this setting.